### DISCUSSION TITLE
Updated Tokens?

### DISCUSSION THREAD

**Discussion by CherryC0p** [fvtt-fria-ligan/twilight2000-foundry-vtt#88] [General]
So the Core Set module has the tokens from the Alpha included. Are those going to be updated to the Full release version?

There are some notable differences, specifically some of the Civilian facings, some of the NATO and Soviet tokens have been changed so there are now 2 of each type, not having the "Local" (Swedish) vehicle tokens, and not having the destroyed vehicle and prone sides for each token. Also the Final release ones are cropped off on the right side, sometimes removing a hand or foot or even the end of a gun barrel.

I've compared to the Release art, which seem lower fidelity than the Alpha ones:

Alpha Token:
![Civ 1](https://user-images.githubusercontent.com/100490835/156013262-d6946946-ad17-4574-841a-aa8b667f7f85.png)

Release Token:
![Civ 1](https://user-images.githubusercontent.com/100490835/156013902-0340af96-5fc6-46aa-a3f6-8104be7f6c9c.png)

**Comment by CherryC0p** [comment]
It actually looks like there are more differences than I originally noticed, so I've updated the previous post. Each faction now has 2 of each type of soldier or vehicle.

Alpha US Soldiers: 1, 6, and 8 are Standing Riflemen, 2, 7, and 10 are Walking Riflemen, 3 is a Grenadier, 4 and 9 are Crouching Riflemen, 5 is a LMG Gunner

Release US Soldiers: 1 and 6 are Standing Riflemen, 2 and 7 are Walking Riflemen, 3 and 8 are Grenadiers, 4 and 9 are Crouching Riflemen, 5 and 10 are LMG Gunners

Alpha US Vehicles: 1-3 are M1 Abrams, 4-6 are M113s, 7-9 are Humvees

Release US Vehicles: 1 and 4 are M1 Abrams, 2 and 5 are Humvees, 3 and 6 are M113s

Alpha USSR: 1 is a LMG Gunner, 2, 7, and 9 are Standing Riflemen, 3, 6, and 8 are Riflemen with their hands up in "Stop" gesture, 4 is a Grenadier, 5 and 10 are Crouching Riflemen

Release USSR: 1 and 6 are LMG Gunners, 2 and 7 are Standing Riflemen, 3 and 8 are "Stop" Riflemen, 4 and 9 are Grenadiers, 5 and 10 are still Crouching Riflemen

Alpha USSR Vehicles: 1-3 are T-72s, 4-6 are BTR70s, 7-9 are UAZ469s

Release USSR: 1 and 4 are T-72s, 2 and 5 are UAZ469s, 3 and 6 are BTR70s

Alpha "Local" Soldiers (Swedish): 1, 6, and 9 are Walking Riflemen with helmets, 2 and 7 are Running Riflemen, 3 is a Grenadier, 4 is a LMG Gunner, 5, 8, and 10 are Standing Riflemen with patrol caps.

Release "Local" Soldiers: 1 and 6 are Walking Riflemen, 2 and 7 are Running Riflemen, 3 and 8 are Grenadiers, 4 and 9 are LMG Gunners and 5 and 10 are Standing Riflemen

No Local Vehicles for Alpha

Release "Local" Vehicles: 1 and 4 are STRV 103Cs, 2 and 5 are TGB 11s, 3 and 6 are STRF 90s

Alpha Civilians: 1 and 6 are the same SMG/Rifle but face opposite directions, same with 2 and 7 (Pistol), 3 and 8 (Spiked Bat), 4 and 9 (Shotgun/Hunting Rifle?), and 5 and 10 (Sawed-off Shotgun/Zip Gun)

Release Civilians: These are numbered the same as Alpha, but they are now facing the same direction, 1 and 6 match, 2 and 7, etc.



On the opposite side (Page 2 of the PDF) the soldiers are in Prone positions and the vehicles are in a destroyed state.

As stated in the previous post, the Full release are less detailed than the Alpha versions, to the point they looks out of focus and parts are cropped out like hands and feet and gun and tank barrels. If there are plans to add the release versions to the module, please see if you can get better versions of the art like you have for the top down vehicles in the Tiles folder.

**Comment by tinwe** [comment]
Yeah, the previous ones sure seems better. I wonder what happened, but I bet Fria Ligan still has more high res and uncropped versions of them that they could update with.

**Comment by CherryC0p** [comment]
there's high res versions of the Vehicles (not as tokens) in the Tiles folder for the module, but nothing for the Soldier tokens. The ones in the Tokens folder are named -vassal-, which leads me to believe they were imported from a homemade Vassal VTT module, not directly from FL. My alpha one that I posted, I use MS paint to crop out of screenshot of the PDF
